projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,68 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: var(--tui-radius-xl);
+    justify-content: center;
+    overflow: hidden;
+
+    > button {
+        .transition(background);
+        position: relative;
+        display: flex;
+
+        border: none;
+        outline: none;
+        background: transparent;
+        align-items: center;
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        flex-direction: column;
+        padding: 1.125rem 0.25rem 1.125rem 0.25rem;
+        gap: 0.5rem;
+        height: auto;
+        font-size: var(--tui-font-text-s);
+        min-width: 7.125rem;
+        max-width: calc(50% - 1.75rem);
+        overflow: visible;
+        cursor: pointer;
+        color: var(--tui-link);
+
+        &:active {
+            background: var(--tui-clear);
+        }
+
+        &:before,
+        &:after {
+            position: absolute;
+            top: 0;
+            background: inherit;
+            width: 1.75rem;
+            height: 100%;
+        }
+
+        &:first-child::before {
+            content: '';
+            left: -1.75rem;
+        }
+
+        &:last-child::after {
+            content: '';
+            right: -1.75rem;
+        }
+    }
+
+    &:has(button:only-child) {
+        border-radius: var(--tui-radius-l);
+    }
+
+    > button:only-child {
+        width: 100%;
+        flex-direction: row;
+        gap: 0.5rem;
```

Comment:
It is the same value, we can drop it from here.

Reply:
fixed